compile-only.sh

```diff
@@ -25,23 +25,19 @@ GRAALVM_VERSION=$(native-image --version)
 echo "Compiling $ARTIFACT with $GRAALVM_VERSION"
 
 native-image \
-  --verbose \
-  --no-fallback \
-  --no-server \
-  --install-exit-handlers \
-  --enable-all-security-services \
-  -Dspring.xml.ignore=false \
-  -H:GenerateDebugInfo=1 \
-  -H:Name=${ARTIFACT} \
-  -H:+PrintClassInitialization \
-  -Dorg.apache.cxf.JDKBugHacks.all=true \
-  -Dspring.native.mode=agent \
-  -H:ConfigurationFileDirectories=./src/main/resources/META-INF/native-image \
-  --initialize-at-build-time=com.sun.xml.bind.v2.runtime.reflect.opt.Injector,com.sun.xml.bind.v2.schemagen.xmlschema \
-  --initialize-at-run-time=org.hibernate.validator.internal.engine.messageinterpolation.el.SimpleELContext \
-  -H:+ReportExceptionStackTraces \
-  -cp ${CP} \
-  ${MAINCLASS}
+    --verbose \
```

Comment:
I've cleaned up the options a bit, leaving only the ones required for the compilation both here and in `compile.sh`. I think the original reason why the configuration file changes weren't properly being picked up is because I ran the `compile-only.sh`, which does not rebuild the JAR files so configuration changes had no effect